content/50_deploying_argocd_to_eks/40_deploy_our_application.md

```diff
@@ -0,0 +1,91 @@
+ +   │  ├─ kubernetes:rbac.authorization.k8s.io:ClusterRole          argocd-server                         created
+ +   │  ├─ kubernetes:rbac.authorization.k8s.io:RoleBinding          argocd/argocd-application-controller  created
+ +   │  ├─ kubernetes:rbac.authorization.k8s.io:Role                 argocd/argocd-dex-server              created
+ +   │  ├─ kubernetes:rbac.authorization.k8s.io:RoleBinding          argocd/argocd-dex-server              created
+ +   │  ├─ kubernetes:rbac.authorization.k8s.io:Role                 argocd/argocd-application-controller  created
+ +   │  ├─ kubernetes:rbac.authorization.k8s.io:RoleBinding          argocd/argocd-server                  created
+ +   │  ├─ kubernetes:rbac.authorization.k8s.io:ClusterRoleBinding   argocd-application-controller         created
+ +   │  ├─ kubernetes:apps:Deployment                                argocd/argocd-server                  created
+ +   │  ├─ kubernetes:apps:Deployment                                argocd/argocd-application-controller  created
+ +   │  ├─ kubernetes:apps:Deployment                                argocd/argocd-dex-server              created
+ +   │  ├─ kubernetes:apps:Deployment                                argocd/argocd-repo-server             created
+ +   │  ├─ kubernetes:apps:Deployment                                argocd/argocd-redis                   created
+ +   │  ├─ kubernetes:apiextensions.k8s.io:CustomResourceDefinition  appprojects.argoproj.io               created     1 warning
+ +   │  └─ kubernetes:apiextensions.k8s.io:CustomResourceDefinition  applications.argoproj.io              created     1 warning
+ +   ├─ pulumi:providers:kubernetes                                  k8s                                   created
+ +   └─ kubernetes:core:Namespace                                    argocd-ns                             created
+
+Outputs:
+    url: "http://ae7c37b7c510511eab4540a6f2211784-521581596.us-west-2.elb.amazonaws.com:80"
+
+Resources:
+    + 34 created
+
+Duration: 1m3s
+
+Permalink: https://app.pulumi.com/workshops/argocd/dev/updates/1
+```
+
+In order to check that everything has been deployed as expected, list the pods in your namespace:
+
+```bash
+kubectl get pods --namespace argocd
+```
+
+And you should see a number of pods:
+
+```
+NAME                                             READY   STATUS    RESTARTS   AGE
+argocd-application-controller-64db95fccd-jkstk   1/1     Running   0          108s
+argocd-dex-server-6b9cbf5598-znfpk               1/1     Running   0          108s
+argocd-redis-6b7b6b766b-dkphm                    1/1     Running   0          107s
+argocd-repo-server-7c9d75b48-tg5r8               1/1     Running   0          107s
+argocd-server-7778cdd5-5f5vd                     1/1     Running   1          109s
+```
+
+The initial password for the ArgoCD server UI is the name of the pod that is running the application. Grab the name from the
```

Comment:
did the bcrypt password setting not work out?

Reply:
Nah - on preview and apply it was generating perpetual diffs as it was re-encrypting each time